platforms/core-execution/build-cache/src/main/java/org/gradle/caching/internal/controller/service/OpFiringLocalBuildCacheServiceHandle.java

```diff
@@ -121,11 +121,11 @@ public long getArchiveSize() {
 
     private static class LocalStoreDetails implements BuildCacheLocalStoreBuildOperationType.Details {
         private final BuildCacheKey key;
-        private final File file;
+        private final long archiveSize;
 
         public LocalStoreDetails(BuildCacheKey key, File file) {
             this.key = key;
-            this.file = file;
```

Comment:
🤔 If we cannot test this on Gradle side, should we add some comment, why we call lenght here?

Reply:
Added.